docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
       - yarn
       - start:dev
   db:
-    image: postgres
+    image: postgres:9.4-alpine
```

Comment:
@samiwel this seems to be causing issues for me, when switching from master -> this branch. Not sure what the cause it, but it spits out a bunch of errors then fails to start. Do you get the same?